sema4ai/vscode-client/src/output/outViewRunIntegration.ts

```diff
@@ -233,3 +235,11 @@ export async function setupDebugSessionOutViewIntegration(context: vscode.Extens
         logError("Error creating socket for Sema4.ai Tasks Output integration.", err, "ROBO_TASKS_SOCKET");
     }
 }
+
+export function applyOutViewIntegrationEnvVars(targetEnv: Record<string, string>) {
```

Comment:
I think that you shouldn't mutate the original entry, rather create a copy here, change the copy and return it (right now from the 2 calls one does the copy and the other one doesn't... rather than updating one of those to create a copy, I think it's better that both can pass the original env without doing a copy and the copy can be done in this function).

Reply:
Yeah, I agree - wanted to keep it as it is, as the original bit implied the mutation - the copy was done "outside" of the additional vars application.

Reply:
done!